### ISSUE TITLE
how to train seecoder?

### ISSUE THREAD

**Issue by pokameng** [SHI-Labs/Prompt-Free-Diffusion#11]

**Comment by pokameng** [comment]
@xingqian2018

**Comment by pokameng** [comment]
hello
@xingqian2018 
I want to know how to train seecoder

**Comment by xingqian2018** [comment]
The details about training the model are in Section 3.3 last paragraph, and Section 4.2 for more detailed hyperparameters. 
In conclusion, it is like Image-Variation training using the common diffusion loss function (variational-low-bounded negative log-likelihood).

We still do not have any plan to release the training code. Need supervisor's approval.
